jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebAppClassLoaderTest.java

```diff
@@ -323,4 +328,23 @@ public void testResources() throws Exception
         
     }
 
+    @Test
+    public void ordering() throws Exception
+    {
+        Enumeration<URL> resources = _loader.getResources("org/acme/clashing.txt");
```

Comment:
Was less confident about relevance or technique of testing `WebInfConfiguration` changes. `_context.getMetaData().getWebInfJars()` in this test is empty.

Reply:
You should be able to create a unit test where you create a WebAppContext, and configure it to look at an unpacked war in src/test/resources. The unpacked war should be minimal, just  needs to have WEB-INF/lib with your alpha and omega jars in it. Then you should be able to call WebInfConfiguration.preConfigure(webapp) and check that _context.getMetaData().getWebInfJars() is in the order you expect.

Reply:
Yes, it seems to work.